docs/app/pages/components/components-sections/facets/dynamic-facets-ng1/snippets/dynamic-facets.snippet.html

```diff
@@ -1,7 +1,15 @@
 <facet-container>
     <!--...-->
     <facet-dynamic name="vm.facetOptions.title" facet-options="vm.facetOptions" display-configuration="vm.facetLongListConfig"
-                   placeholder=" 'Find Author' " update-callback="vm.longListCallback" max-length="100" max-lines="3" disable-return="'false'" >
+                   placeholder=" 'Find Author' " update-callback="vm.longListCallback" max-length="100" max-lines="3" disable-return="'false'"
+                   facet-option-template="'options-with-star'" >
     </facet-dynamic>
     <!--...-->
-</facet-container>
\ No newline at end of file
+</facet-container>
+
+<script type="text/ng-template" id="options-with-star">
```

Comment:
We usually document templates in a separate tab, partly because the syntax highlighter doesn't handle HTML in script tags properly.

Reply:
Done